starters/express-typescript-typeorm-postgres/src/utils/log-helper.spec.ts

```diff
@@ -0,0 +1,153 @@
+describe(`LogHelper`, () => {
+  let consoleSpy;
+  beforeEach(() => {
+    jest.resetModules();
+  });
+
+  afterEach(() => {
+    jest.resetAllMocks();
+  });
+
+  describe(`log method`, () => {
+    beforeEach(() => {
+      consoleSpy = jest.spyOn(console, 'log');
+    });
+
+    it(`does not call console.log`, () => {
+      process.env.ALLOWED_LOG_LEVELS = `["ERROR"]`;
+      // eslint-disable-next-line @typescript-eslint/no-var-requires
+      const { LogHelper } = require('./log-helper');
```

Comment:
We should probably use `await import('./log-helper')` instead of `require()` for consistency and a modern approach

Reply:
ok